qiskit/result/result.py

```diff
@@ -14,19 +14,19 @@
 
 """Model for schema-conformant Results."""
 
+import copy
+
 from qiskit.circuit.quantumcircuit import QuantumCircuit
 from qiskit.pulse.schedule import Schedule
 from qiskit.exceptions import QiskitError
 from qiskit.quantum_info.states import Statevector
-
-from qiskit.validation.base import BaseModel, bind_schema
+from qiskit.result.models import ExperimentResult
 from qiskit.result import postprocess
 from qiskit.qobj.utils import MeasLevel
-from .models import ResultSchema
+from qiskit.qobj import QobjHeader
 
 
-@bind_schema(ResultSchema)
-class Result(BaseModel):
+class Result:
     """Model for Results.
 
     Please note that this class only describes the required fields. For the
```

Comment:
Lines 32-33 are probably no longer needed as there is no `ResultSchema` anymore.

Reply:
Yep good catch, I meant to update this

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/4030/commits/514d58fef10d2e3db29bfe4c7a71fb1ddf121c9a